2019-04-24 14:12:49,303 - ERROR - app.py - to_int - 195 - invalid literal for int() with base 10: ''
2019-04-24 14:16:48,502 - ERROR - app.py - to_int - 195 - 修改规则时输入格式错误
2019-04-24 14:16:48,503 - ERROR - app.py - to_int - 196 - invalid literal for int() with base 10: ''
2019-04-24 14:19:16,521 - ERROR - app.py - to_int - 195 - 修改规则时输入格式错误
2019-04-24 14:19:16,522 - ERROR - app.py - to_int - 196 - invalid literal for int() with base 10: ''
2019-04-24 14:26:32,396 - INFO - app.py - edit_rule - 229 - {"id": 1, "minRate": "0.0", "maxRate": "0.0", "maxFlightDistance": 0, "maxFlightTime": "", "maxStopPoint": "0", "maxTransfer": "0", "permission": ["DDD"], "prohibition": ["FFF"], "yesSpanCity": ["TRU"], "notSpanCity": ["ADF"], "maxStrideDays": 0, "banAllDays": 0, "banType": ["string"], "carrierWhite": ["string"], "carrierBlack": ["string"], "legTransferWhite": ["string"], "transferBlack": ["ING"], "legTransferBlack": ["string"]}
2019-04-24 14:26:32,397 - INFO - app.py - edit_rule - 230 - 修改规则成功
2019-04-24 14:46:20,153 - INFO - app.py - edit_rule - {"id": 1, "minRate": "0.0", "maxRate": "0.0", "maxFlightDistance": 0, "maxFlightTime": "", "maxStopPoint": "0", "maxTransfer": "0", "permission": ["DDD"], "prohibition": ["FFF"], "yesSpanCity": ["TRU"], "notSpanCity": ["ADF"], "maxStrideDays": 0, "banAllDays": 0, "banType": ["string"], "carrierWhite": ["string"], "carrierBlack": ["string"], "legTransferWhite": ["string"], "transferBlack": ["ING"], "legTransferBlack": ["string"]}
2019-04-24 14:46:20,154 - INFO - app.py - edit_rule - 修改规则成功
2019-04-24 14:48:21,955 - INFO - app.py - edit_rule - {"id": 1, "minRate": "0.0", "maxRate": "0.0", "maxFlightDistance": 0, "maxFlightTime": "", "maxStopPoint": "0", "maxTransfer": "0", "permission": ["DDD"], "prohibition": ["FFF"], "yesSpanCity": ["TRU"], "notSpanCity": ["ADF"], "maxStrideDays": 0, "banAllDays": 0, "banType": ["string"], "carrierWhite": ["string"], "carrierBlack": ["string"], "legTransferWhite": ["string"], "transferBlack": ["ING"], "legTransferBlack": ["string"]}
2019-04-24 14:48:21,956 - INFO - app.py - edit_rule - 修改规则成功
2019-04-24 14:49:04,040 - INFO - app.py - edit_rule - {"id": 1, "minRate": "0.0", "maxRate": "0.0", "maxFlightDistance": 0, "maxFlightTime": "", "maxStopPoint": "0", "maxTransfer": "0", "permission": ["DDD", "AAA"], "prohibition": ["FFF"], "yesSpanCity": ["TRU"], "notSpanCity": ["ADF"], "maxStrideDays": 0, "banAllDays": 0, "banType": ["string"], "carrierWhite": ["string"], "carrierBlack": ["string"], "legTransferWhite": ["string"], "transferBlack": ["ING"], "legTransferBlack": ["string"]}
2019-04-24 14:49:04,041 - INFO - app.py - edit_rule - 修改规则成功
2019-04-24 15:29:38,470 - ERROR - app.py - edit_rule - 修改规则失败
2019-04-24 15:34:23,804 - ERROR - app.py - edit_rule - 修改规则失败
2019-04-24 15:55:44,852 - ERROR - app.py - edit_rule - 修改规则失败
2019-04-24 16:01:37,098 - ERROR - app.py - log_exception - Exception on /edit_rule/2 [POST]
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/xiyanzhi/workspace/myflaskapp/app.py", line 122, in wrap
    return f(*args, **kwargs)
  File "/Users/xiyanzhi/workspace/myflaskapp/app.py", line 225, in edit_rule
    dict = processInput(request.form.to_dict())
  File "/Users/xiyanzhi/workspace/myflaskapp/app.py", line 165, in processInput
    dict["banType"] = str_split(dict["banType"])
  File "/Users/xiyanzhi/workspace/myflaskapp/app.py", line 153, in str_split
    return str.split('')
ValueError: empty separator
2019-04-24 16:03:43,821 - ERROR - app.py - edit_rule - 修改规则失败
2019-04-24 16:04:08,201 - ERROR - app.py - edit_rule - 修改规则失败
2019-04-24 16:07:33,513 - ERROR - app.py - edit_rule - 修改规则失败
2019-04-24 16:10:42,688 - INFO - app.py - edit_rule - {"id": 2, "minRate": "1.1", "maxRate": "1", "maxFlightDistance": 1, "maxFlightTime": 1, "maxStopPoint": "1", "maxTransfer": "1", "permission": ["1"], "prohibition": ["1"], "yesSpanCity": ["CAN", "PEK", "CAN", "PEK"], "notSpanCity": ["CAN", "PEK"], "maxStrideDays": 1, "banAllDays": 1, "banType": ["CAN", "PEK"], "carrierWhite": ["CAN", "PEK"], "carrierBlack": ["CAN-PEK"], "legTransferWhite": ["CAN-PEK\uff1aWUH", "CAN-PEK\uff1aWUH"], "transferBlack": ["1"], "legTransferBlack": ["CAN-PEK\uff1aWUH", "CAN-PEK\uff1aWUH"]}
2019-04-24 16:10:42,688 - INFO - app.py - edit_rule - 修改规则成功
2019-04-24 16:13:49,838 - INFO - app.py - edit_rule - {"id": 2, "minRate": "1.1", "maxRate": "1.0", "maxFlightDistance": 1, "maxFlightTime": "", "maxStopPoint": "1", "maxTransfer": "1", "permission": ["1"], "prohibition": ["1"], "yesSpanCity": ["CAN", "", "", "PEK", "", "", "CAN", "", "", "PEK"], "notSpanCity": ["CAN", "", "", "PEK"], "maxStrideDays": 1, "banAllDays": 1, "banType": ["CAN", "", "", "PEK"], "carrierWhite": ["CAN", "", "", "PEK"], "carrierBlack": ["CAN-PEK"], "legTransferWhite": ["CAN-PEK:WUH", "", "", "CAN-PEK:WUH"], "transferBlack": ["1"], "legTransferBlack": ["CAN-PEK:WUH", "", "", "CAN-PEK:WUH"]}
2019-04-24 16:13:49,838 - INFO - app.py - edit_rule - 修改规则成功
2019-04-24 16:16:28,149 - INFO - app.py - edit_rule - {"id": 2, "minRate": "1.1", "maxRate": "1.0", "maxFlightDistance": 1, "maxFlightTime": "", "maxStopPoint": "1", "maxTransfer": "1", "permission": ["1"], "prohibition": ["1"], "yesSpanCity": ["CAN", "", "", "PEK", "", "", "CAN", "", "", "PEK"], "notSpanCity": ["CAN", "", "", "PEK"], "maxStrideDays": 1, "banAllDays": 1, "banType": ["CAN", "", "", "PEK"], "carrierWhite": ["CAN", "", "", "PEK"], "carrierBlack": ["CAN-PEK"], "legTransferWhite": ["CAN-PEK:WUH", "", "", "CAN-PEK:WUH"], "transferBlack": ["1"], "legTransferBlack": ["CAN-PEK:WUH", "", "", "CAN-PEK:WUH"]}
2019-04-24 16:16:28,149 - INFO - app.py - edit_rule - 修改规则成功
2019-04-24 16:17:25,901 - INFO - app.py - edit_rule - {"id": 2, "minRate": "1.1", "maxRate": "1.0", "maxFlightDistance": 1, "maxFlightTime": "", "maxStopPoint": "1", "maxTransfer": "1", "permission": ["1"], "prohibition": ["1"], "yesSpanCity": ["CAN", "PEK", "CAN", "PEK"], "notSpanCity": ["CAN", "PEK"], "maxStrideDays": 1, "banAllDays": 1, "banType": ["CAN", "PEK"], "carrierWhite": ["CAN", "PEK"], "carrierBlack": ["CAN-PEK"], "legTransferWhite": ["CAN-PEK:WUH", "CAN-PEK:WUH"], "transferBlack": ["1"], "legTransferBlack": ["CAN-PEK:WUH", "CAN-PEK:WUH"]}
2019-04-24 16:17:25,902 - INFO - app.py - edit_rule - 修改规则成功
2019-04-24 16:29:44,258 - INFO - app.py - edit_rule - {"id": 2, "minRate": "0.1", "maxRate": "0.0", "maxFlightDistance": 0, "maxFlightTime": "", "maxStopPoint": "0", "maxTransfer": "2", "permission": ["string"], "prohibition": ["string"], "yesSpanCity": ["string"], "notSpanCity": ["string"], "maxStrideDays": 0, "banAllDays": 0, "banType": ["string"], "carrierWhite": ["string"], "carrierBlack": ["string"], "legTransferWhite": ["string"], "transferBlack": ["string"], "legTransferBlack": ["string"]}
2019-04-24 16:29:44,260 - INFO - app.py - edit_rule - 修改规则成功
2019-04-24 16:30:45,558 - ERROR - app.py - add_rule - 添加规则失败
2019-04-24 16:40:15,396 - ERROR - app.py - log_exception - Exception on /add_rule [POST]
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/xiyanzhi/workspace/myflaskapp/app.py", line 122, in wrap
    return f(*args, **kwargs)
  File "/Users/xiyanzhi/workspace/myflaskapp/app.py", line 206, in add_rule
    dict = processInput(request.form.to_dict())
  File "/Users/xiyanzhi/workspace/myflaskapp/app.py", line 178, in processInput
    removed = dict(x for x in dict.iteritems() if all(x))
TypeError: 'dict' object is not callable
2019-04-24 16:44:07,935 - ERROR - app.py - add_rule - 添加规则失败
2019-04-25 09:58:27,831 - ERROR - app.py - dashboard - 连接数据库失败
2019-04-25 09:58:46,597 - ERROR - app.py - dashboard - 连接数据库失败
2019-04-25 10:00:47,251 - ERROR - app.py - dashboard - 连接数据库失败
2019-04-25 10:01:39,796 - ERROR - app.py - dashboard - 连接数据库失败
2019-04-25 10:01:41,281 - ERROR - app.py - dashboard - 连接数据库失败
2019-04-25 10:01:51,213 - ERROR - app.py - dashboard - 连接数据库失败
2019-04-25 10:02:21,240 - ERROR - app.py - dashboard - 连接数据库失败
2019-04-25 10:02:22,259 - ERROR - app.py - dashboard - 连接数据库失败
2019-04-25 10:02:39,415 - ERROR - app.py - dashboard - 连接数据库失败
2019-04-25 10:02:57,743 - ERROR - app.py - dashboard - 连接数据库失败
2019-04-25 10:03:01,080 - ERROR - app.py - dashboard - 连接数据库失败
2019-04-25 10:03:58,353 - ERROR - app.py - dashboard - 连接数据库失败
2019-04-25 10:05:12,374 - ERROR - app.py - dashboard - 连接数据库失败
2019-04-25 10:06:38,207 - ERROR - app.py - dashboard - 连接数据库失败
2019-04-25 10:08:46,381 - INFO - app.py - edit_rule - {"notSpanCity": ["notSpanCity"], "maxStopPoint": "0", "permission": ["permission"], "maxTransfer": "0", "minRate": "0.0", "prohibition": ["prohibition"], "carrierWhite": ["CZ"], "legTransferBlack": ["legTransferBlack"], "maxRate": "0.0", "banType": ["banType"], "legTransferWhite": ["legTransferWhite"], "carrierBlack": ["carrierBlack"], "yesSpanCity": ["yesSpanCity"], "id": 1, "transferBlack": ["transferBlack"]}
2019-04-25 10:08:46,382 - INFO - app.py - edit_rule - 修改规则成功
2019-04-25 10:08:55,045 - INFO - app.py - edit_rule - {"notSpanCity": ["notSpanCity"], "maxStopPoint": "0", "permission": ["permission"], "maxTransfer": "0", "minRate": "0.0", "prohibition": ["prohibition"], "carrierWhite": ["GZ"], "legTransferBlack": ["legTransferBlack"], "maxRate": "0.0", "banType": ["banType"], "legTransferWhite": ["legTransferWhite"], "carrierBlack": ["carrierBlack"], "yesSpanCity": ["yesSpanCity"], "id": 1, "transferBlack": ["transferBlack"]}
2019-04-25 10:08:55,046 - INFO - app.py - edit_rule - 修改规则成功
2019-04-25 10:09:40,401 - INFO - app.py - delete_rule - delete rule id = 1
2019-04-25 10:09:40,402 - INFO - app.py - delete_rule - 删除规则成功
2019-04-25 10:09:45,573 - INFO - app.py - delete_rule - delete rule id = 1
2019-04-25 10:09:45,574 - INFO - app.py - delete_rule - 删除规则成功
2019-04-25 10:11:26,324 - INFO - app.py - edit_rule - {"id": 1, "minRate": "0.0", "maxRate": "0.0", "maxStopPoint": "0", "maxTransfer": "0", "permission": ["permission"], "prohibition": ["prohibition"], "yesSpanCity": ["yesSpanCity"], "notSpanCity": ["notSpanCity"], "banType": ["banType"], "transferBlack": ["transferBlack"], "carrierWhite": ["sz"], "carrierBlack": ["carrierBlack"], "legTransferWhite": ["legTransferWhite"], "legTransferBlack": ["legTransferBlack"]}
2019-04-25 10:11:26,324 - INFO - app.py - edit_rule - 修改规则成功
2019-04-25 10:11:37,999 - INFO - app.py - edit_rule - {"id": 2, "minRate": "1.1", "maxRate": "0.0", "maxStopPoint": "0", "maxTransfer": "0", "permission": ["string"], "prohibition": ["string"], "yesSpanCity": ["string"], "notSpanCity": ["string"], "banType": ["string"], "transferBlack": ["string"], "carrierWhite": ["string"], "carrierBlack": ["string"], "legTransferWhite": ["string"], "legTransferBlack": ["string"]}
2019-04-25 10:11:37,999 - INFO - app.py - edit_rule - 修改规则成功
2019-04-25 10:12:07,405 - INFO - app.py - add_rule - {"minRate": "7"}
2019-04-25 10:12:07,405 - INFO - app.py - add_rule - 添加规则成功
2019-04-25 10:20:37,601 - INFO - app.py - edit_rule - {"id": 1, "minRate": "0.0", "maxRate": "0.0", "maxStopPoint": "0", "maxTransfer": "0", "permission": ["permission"], "prohibition": ["prohibition"], "yesSpanCity": ["yesSpanCity"], "notSpanCity": ["notSpanCity"], "banType": ["banType"], "transferBlack": ["transferBlack"], "carrierWhite": ["sz"], "carrierBlack": ["carrierBlack", "GZ"], "legTransferWhite": ["legTransferWhite"], "legTransferBlack": ["legTransferBlack"]}
2019-04-25 10:20:37,603 - INFO - app.py - edit_rule - 修改规则成功
2019-04-25 10:30:52,321 - INFO - app.py - add_rule - {"minRate": "1.7"}
2019-04-25 10:30:52,322 - INFO - app.py - add_rule - 添加规则成功
2019-04-25 10:30:59,755 - INFO - app.py - delete_rule - delete rule id = 21
2019-04-25 10:30:59,755 - INFO - app.py - delete_rule - 删除规则成功
2019-04-25 10:31:06,995 - INFO - app.py - delete_rule - delete rule id = 21
2019-04-25 10:31:06,996 - INFO - app.py - delete_rule - 删除规则成功
2019-04-25 10:36:27,604 - ERROR - app.py - delete_rule - 删除规则失败
2019-04-25 10:36:31,965 - ERROR - app.py - delete_rule - 删除规则失败
2019-04-25 10:36:46,759 - ERROR - app.py - delete_rule - 删除规则失败
2019-04-25 10:38:44,266 - INFO - app.py - add_rule - {"minRate": "2.2"}
2019-04-25 10:38:44,267 - INFO - app.py - add_rule - 添加规则成功
2019-04-25 10:43:46,975 - ERROR - app.py - add_rule - 添加规则失败
2019-04-25 10:44:00,499 - ERROR - app.py - add_rule - 添加规则失败
2019-04-25 10:44:29,980 - INFO - app.py - add_rule - {"carrierWhite": ["3"]}
2019-04-25 10:44:29,981 - INFO - app.py - add_rule - 添加规则成功
2019-04-25 10:45:06,758 - INFO - app.py - add_rule - {"maxRate": "10", "carrierBlack": ["3"]}
2019-04-25 10:45:06,759 - INFO - app.py - add_rule - 添加规则成功
2019-04-25 11:37:50,140 - INFO - app.py - edit_rule - {"id": 1, "minRate": "0.0", "maxRate": "0.0", "maxStopPoint": "0", "maxTransfer": "0", "permission": ["permission"], "prohibition": ["prohibition"], "yesSpanCity": ["yesSpanCity"], "notSpanCity": ["notSpanCity"], "banType": ["banType"], "transferBlack": ["transferBlack"], "carrierWhite": ["CZ"], "carrierBlack": ["carrierBlack", "GZ"], "legTransferWhite": ["legTransferWhite"], "legTransferBlack": ["legTransferBlack"]}
2019-04-25 11:37:50,142 - INFO - app.py - edit_rule - 修改规则成功
2019-04-25 11:37:59,755 - ERROR - app.py - delete_rule - 删除规则失败
2019-04-25 11:38:14,891 - INFO - app.py - add_rule - {"minRate": "1.8"}
2019-04-25 11:38:14,892 - INFO - app.py - add_rule - 添加规则成功
2019-04-25 11:39:26,728 - INFO - app.py - add_rule - {"carrierBlack": ["XXX"]}
2019-04-25 11:39:26,729 - INFO - app.py - add_rule - 添加规则成功
2019-04-25 14:41:56,096 - ERROR - app.py - delete_rule - 删除规则失败
2019-04-25 14:42:56,392 - ERROR - app.py - delete_rule - 删除规则失败
2019-04-25 14:43:35,993 - ERROR - app.py - delete_rule - 删除规则失败
2019-04-25 14:43:53,138 - ERROR - app.py - delete_rule - 删除规则失败
2019-04-25 15:20:06,384 - ERROR - app.py - dashboard - 连接数据库失败
2019-04-25 15:20:15,998 - ERROR - app.py - dashboard - 连接数据库失败
2019-04-25 15:20:55,186 - ERROR - app.py - dashboard - 连接数据库失败
2019-04-25 16:14:44,087 - INFO - app.py - edit_rule - {"id": 2, "minRate": "1.1", "maxRate": "0.0", "maxStopPoint": "0", "maxTransfer": "0", "permission": ["string"], "prohibition": ["string"], "yesSpanCity": ["string"], "notSpanCity": ["string"], "banAllDays": 1, "banType": ["string"], "transferBlack": ["string"], "carrierWhite": ["string"], "carrierBlack": ["string"], "legTransferWhite": ["string"], "legTransferBlack": ["string"]}
2019-04-25 16:14:44,089 - INFO - app.py - edit_rule - 修改规则成功
2019-04-25 16:15:30,253 - INFO - app.py - edit_rule - {"id": 2, "minRate": "1.1", "maxRate": "0.0", "maxStopPoint": "0", "maxTransfer": "0", "permission": ["string"], "prohibition": ["string"], "yesSpanCity": ["string"], "notSpanCity": ["string"], "banType": ["string"], "transferBlack": ["string"], "carrierWhite": ["string"], "carrierBlack": ["string"], "legTransferWhite": ["string"], "legTransferBlack": ["string"]}
2019-04-25 16:15:30,254 - INFO - app.py - edit_rule - 修改规则成功
2019-04-25 16:36:13,790 - ERROR - app.py - processInput - 修改规则时输入格式错误
2019-04-25 16:36:13,793 - ERROR - app.py - processInput - 'banAllDays'
2019-04-25 16:38:29,568 - ERROR - app.py - processInput - 修改规则时输入格式错误
2019-04-25 16:38:29,569 - ERROR - app.py - processInput - 'banAllDays'
2019-04-25 16:38:38,809 - ERROR - app.py - processInput - 修改规则时输入格式错误
2019-04-25 16:38:38,812 - ERROR - app.py - processInput - 'banAllDays'
2019-04-25 16:41:25,086 - ERROR - app.py - processInput - 修改规则时输入格式错误
2019-04-25 16:41:25,086 - ERROR - app.py - processInput - 'banAllDays'
2019-04-25 16:41:42,504 - ERROR - app.py - processInput - 修改规则时输入格式错误
2019-04-25 16:41:42,504 - ERROR - app.py - processInput - 'banAllDays'
2019-04-25 16:45:12,320 - ERROR - app.py - processInput - 修改规则时输入格式错误
2019-04-25 16:45:12,322 - ERROR - app.py - processInput - u'banAllDays'
2019-04-25 16:46:48,607 - ERROR - app.py - processInput - 修改规则时输入格式错误
2019-04-25 16:46:48,609 - ERROR - app.py - processInput - u'banAllDays'
2019-04-25 16:46:48,621 - ERROR - app.py - log_exception - Exception on /edit_rule/3 [POST]
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/xiyanzhi/workspace/myflaskapp/app.py", line 122, in wrap
    return f(*args, **kwargs)
  File "/Users/xiyanzhi/workspace/myflaskapp/app.py", line 251, in edit_rule
    form = json.dumps(dict)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 243, in dumps
    return _default_encoder.encode(obj)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: <Response 227 bytes [302 FOUND]> is not JSON serializable
2019-04-25 16:47:28,990 - ERROR - app.py - processInput - 修改规则时输入格式错误
2019-04-25 16:47:28,991 - ERROR - app.py - processInput - u'banAllDays'
2019-04-25 16:48:29,176 - ERROR - app.py - processInput - 修改规则时输入格式错误
2019-04-25 16:48:29,180 - ERROR - app.py - processInput - u'banAllDays'
2019-04-25 16:53:30,623 - ERROR - app.py - processInput - 修改规则时输入格式错误
2019-04-25 16:56:34,377 - ERROR - app.py - processInput - 修改规则时部分输入为空
2019-04-25 16:56:34,382 - ERROR - app.py - processInput - u'banAllDays'
2019-04-25 17:14:55,928 - ERROR - app.py - to_int - 修改规则时输入格式错误
2019-04-25 17:14:55,929 - ERROR - app.py - to_int - invalid literal for int() with base 10: "''"
2019-04-25 17:14:55,955 - INFO - app.py - edit_rule - {"id": 2, "minRate": "1.1", "maxRate": "0.0", "maxStopPoint": "0", "maxTransfer": "0", "permission": ["string"], "prohibition": ["string"], "yesSpanCity": ["string"], "notSpanCity": ["string"], "banType": ["string"], "transferBlack": ["string"], "carrierWhite": ["string"], "carrierBlack": ["string"], "legTransferWhite": ["string"], "legTransferBlack": ["string"]}
2019-04-25 17:14:55,956 - INFO - app.py - edit_rule - 修改规则成功
2019-04-25 17:16:44,942 - ERROR - app.py - to_int - 修改规则时输入格式错误
2019-04-25 17:18:13,243 - ERROR - app.py - to_int - 修改规则时输入格式错误
2019-04-25 17:18:13,245 - ERROR - app.py - to_int - invalid literal for int() with base 10: "''"
2019-04-25 17:23:38,626 - INFO - app.py - edit_rule - {"notSpanCity": ["string"], "maxStopPoint": "0", "permission": ["string"], "maxTransfer": "0", "minRate": "1.1", "prohibition": ["string"], "carrierWhite": ["string"], "legTransferBlack": ["string"], "maxRate": "0.0", "banType": ["string"], "legTransferWhite": ["string"], "carrierBlack": ["string"], "yesSpanCity": ["string"], "id": 2, "transferBlack": ["string"]}
2019-04-25 17:23:38,628 - INFO - app.py - edit_rule - 修改规则成功
2019-04-25 17:25:00,830 - INFO - app.py - edit_rule - {"notSpanCity": ["string"], "maxStopPoint": "0", "permission": ["string"], "maxTransfer": "0", "minRate": "1.1", "prohibition": ["string"], "carrierWhite": ["string"], "legTransferBlack": ["string"], "maxRate": "0.0", "banType": ["string"], "legTransferWhite": ["string"], "carrierBlack": ["string"], "yesSpanCity": ["string"], "id": 2, "transferBlack": ["string"]}
2019-04-25 17:25:00,833 - INFO - app.py - edit_rule - 修改规则成功
2019-04-25 17:28:36,879 - ERROR - app.py - log_exception - Exception on /dashboard [GET]
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Library/Python/2.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/xiyanzhi/workspace/myflaskapp/app.py", line 122, in wrap
    return f(*args, **kwargs)
  File "/Users/xiyanzhi/workspace/myflaskapp/app.py", line 214, in dashboard
    return render_template('dashboard.html', rules=rules)
  File "/Library/Python/2.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Library/Python/2.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Library/Python/2.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Library/Python/2.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Library/Python/2.7/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/Library/Python/2.7/site-packages/jinja2/environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/Library/Python/2.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/xiyanzhi/workspace/myflaskapp/templates/dashboard.html", line 377, in template
    <td>{{是}}</td>
TemplateSyntaxError: unexpected char u'\u662f' at 9084
2019-04-25 17:41:36,161 - INFO - app.py - edit_rule - {"notSpanCity": ["string"], "maxStopPoint": "0", "permission": ["string"], "maxTransfer": "0", "minRate": "1.1", "prohibition": ["string"], "carrierWhite": ["string"], "legTransferBlack": ["string"], "maxRate": "0.0", "banType": ["string"], "legTransferWhite": ["string"], "carrierBlack": ["string"], "yesSpanCity": ["string"], "id": 2, "transferBlack": ["string"]}
2019-04-25 17:41:36,163 - INFO - app.py - edit_rule - 修改规则成功
2019-04-25 17:50:07,085 - INFO - app.py - edit_rule - {"id": 2, "minRate": "1.1", "maxRate": "0.0", "maxStopPoint": "0", "maxTransfer": "0", "permission": ["string"], "prohibition": ["string"], "yesSpanCity": ["string"], "notSpanCity": ["string"], "banType": ["string"], "transferBlack": ["string"], "carrierWhite": ["string"], "carrierBlack": ["string"], "legTransferWhite": ["string"], "legTransferBlack": ["string"]}
2019-04-25 17:50:07,086 - INFO - app.py - edit_rule - 修改规则成功
2019-04-25 17:50:54,632 - INFO - app.py - edit_rule - {"id": 2, "minRate": "1.1", "maxRate": "0.0", "maxStopPoint": "0", "maxTransfer": "0", "permission": ["string"], "prohibition": ["string"], "yesSpanCity": ["string"], "notSpanCity": ["string"], "banType": ["string"], "transferBlack": ["string"], "carrierWhite": ["string"], "carrierBlack": ["string"], "legTransferWhite": ["string"], "legTransferBlack": ["string"]}
2019-04-25 17:50:54,633 - INFO - app.py - edit_rule - 修改规则成功
2019-04-25 17:51:56,991 - INFO - app.py - edit_rule - {"id": 2, "minRate": "1.1", "maxRate": "0.0", "maxStopPoint": "0", "maxTransfer": "0", "permission": ["string"], "prohibition": ["string"], "yesSpanCity": ["string"], "notSpanCity": ["string"], "banType": ["string"], "transferBlack": ["string"], "carrierWhite": ["string"], "carrierBlack": ["string"], "legTransferWhite": ["string"], "legTransferBlack": ["string"]}
2019-04-25 17:51:56,992 - INFO - app.py - edit_rule - 修改规则成功
2019-04-25 17:53:08,716 - INFO - app.py - edit_rule - {"id": 2, "minRate": "1.1", "maxRate": "0.0", "maxStopPoint": "0", "maxTransfer": "0", "permission": ["string"], "prohibition": ["string"], "yesSpanCity": ["string"], "notSpanCity": ["string"], "banType": ["string"], "transferBlack": ["string"], "carrierWhite": ["string"], "carrierBlack": ["string"], "legTransferWhite": ["string"], "legTransferBlack": ["string"]}
2019-04-25 17:53:08,717 - INFO - app.py - edit_rule - 修改规则成功
2019-04-25 17:57:08,172 - INFO - app.py - edit_rule - {"id": 2, "minRate": "1.1", "maxRate": "0.0", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "permission": ["string"], "prohibition": ["string"], "yesSpanCity": ["string"], "notSpanCity": ["string"], "maxStrideDays": 0, "banAllDays": 0, "banType": ["string"], "transferBlack": ["string"], "carrierWhite": ["string"], "carrierBlack": ["string"], "legTransferWhite": ["string"], "legTransferBlack": ["string"]}
2019-04-25 17:57:08,172 - INFO - app.py - edit_rule - 修改规则成功
2019-04-25 18:01:25,561 - INFO - app.py - add_rule - {"banAllDays": 1}
2019-04-25 18:01:25,562 - INFO - app.py - add_rule - 添加规则成功
2019-04-25 18:01:35,445 - INFO - app.py - add_rule - {"prohibition": ["stringer"], "banAllDays": 1}
2019-04-25 18:01:35,445 - INFO - app.py - add_rule - 添加规则成功
2019-04-25 18:06:08,836 - INFO - app.py - add_rule - {"transferBlack": ["GG"]}
2019-04-25 18:06:08,837 - INFO - app.py - add_rule - 添加规则成功
2019-04-26 14:40:02,016 - ERROR - app.py - delete_rule - 删除规则失败
2019-04-26 14:40:02,723 - ERROR - app.py - delete_rule - 删除规则失败
2019-04-26 14:40:03,587 - ERROR - app.py - delete_rule - 删除规则失败
2019-04-26 14:40:10,127 - ERROR - app.py - delete_rule - 删除规则失败
2019-04-26 14:58:38,206 - INFO - app.py - add_rule - {"banAllDays": 0}
2019-04-26 14:58:38,207 - INFO - app.py - add_rule - 添加规则成功
2019-04-26 14:59:49,652 - INFO - app.py - add_rule - {"banAllDays": 0, "legTranferWhite": "ACX"}
2019-04-26 14:59:49,654 - INFO - app.py - add_rule - 添加规则成功
2019-04-26 15:02:15,924 - ERROR - app.py - add_rule - 添加规则失败
2019-04-28 17:29:05,636 - INFO - app.py - edit_rule - {"id": 65, "minRate": "0.0", "maxRate": "0.0", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxStrideDays": 0, "banAllDays": 0, "legTransferWhite": ["A-B:C", "D-E:F"], "type": 0}
2019-04-28 17:29:05,638 - INFO - app.py - edit_rule - 修改规则成功
2019-04-28 18:21:13,174 - INFO - app.py - add_rule - {"type": 0, "yesSpanCity": ["AKU"], "banAllDays": 0}
2019-04-28 18:21:13,176 - INFO - app.py - add_rule - 添加规则成功
2019-05-07 14:10:58,980 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-07 14:11:05,998 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-07 14:13:36,095 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-07 15:09:44,302 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-07 15:13:16,144 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-07 15:13:34,207 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-07 15:13:35,918 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-07 15:16:52,923 - INFO - app.py - add_rule - {"type": 0, "banAllDays": 0, "banType": ["E90"]}
2019-05-07 15:16:52,924 - INFO - app.py - add_rule - 添加规则成功
2019-05-07 15:39:23,898 - INFO - app.py - add_rule - {"type": 0, "banAllDays": 0, "transferBlack": ["CMB"]}
2019-05-07 15:39:23,900 - INFO - app.py - add_rule - 添加规则成功
2019-05-07 16:02:17,573 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-07 16:04:31,476 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-07 16:04:32,897 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-07 16:06:36,544 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-07 16:11:25,155 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-07 16:11:38,482 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-07 16:12:15,039 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-07 16:18:46,837 - INFO - app.py - add_rule - {"type": 0, "maxTransferTime": "222", "banAllDays": 0}
2019-05-07 16:18:46,838 - INFO - app.py - add_rule - 添加规则成功
2019-05-07 16:19:01,080 - INFO - app.py - add_rule - {"type": 0, "yesSpanCity": ["ADL"], "banAllDays": 0}
2019-05-07 16:19:01,080 - INFO - app.py - add_rule - 添加规则成功
2019-05-07 16:44:58,538 - INFO - app.py - add_rule - {"type": 0, "maxTransferTime": "100", "banAllDays": 0}
2019-05-07 16:44:58,539 - INFO - app.py - add_rule - 添加规则成功
2019-05-07 17:34:04,663 - INFO - app.py - edit_rule - {"id": 61, "minRate": "0.0", "maxRate": "0.0", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "111", "maxStrideDays": 0, "banAllDays": 0, "banType": ["E90", "7M8", "73H", "333"], "transferBlack": ["AKL", "AKU", "AMS", "AOG", "ADL"], "carrierWhite": ["AF", "SV"], "carrierBlack": ["WS", "AF"], "type": 0}
2019-05-07 17:34:04,667 - INFO - app.py - edit_rule - 修改规则成功
2019-05-07 17:35:30,800 - INFO - app.py - edit_rule - {"id": 62, "minRate": "1.2", "maxRate": "0.0", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-07 17:35:30,801 - INFO - app.py - edit_rule - 修改规则成功
2019-05-08 16:27:43,098 - INFO - app.py - edit_rule - {"id": 61, "minRate": "1.2", "maxRate": "0.0", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "111", "maxStrideDays": 0, "banAllDays": 0, "banType": ["E90", "7M8", "73H", "333"], "transferBlack": ["AKL", "AKU", "AMS", "AOG", "ADL"], "carrierWhite": ["AF", "SV"], "carrierBlack": ["WS", "AF"], "type": 0}
2019-05-08 16:27:43,106 - INFO - app.py - edit_rule - 修改规则成功
2019-05-08 17:54:00,289 - ERROR - app.py - log_exception - Exception on /edit_rule/61 [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 380, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1321, in getresponse
    response.begin()
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 296, in begin
    version, status, reason = self._read_status()
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 265, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/requests/adapters.py", line 449, in send
    timeout=timeout
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 638, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/usr/local/lib/python3.7/site-packages/urllib3/util/retry.py", line 368, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/usr/local/lib/python3.7/site-packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 380, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1321, in getresponse
    response.begin()
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 296, in begin
    version, status, reason = self._read_status()
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 265, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/xiyanzhi/workspace/myflaskapp/app.py", line 108, in wrap
    return f(*args, **kwargs)
  File "/Users/xiyanzhi/workspace/myflaskapp/app.py", line 264, in edit_rule
    r = requests.put(app.config['ENDPOINT']+app.config['UPDATE_URL'], headers=header, data=form)
  File "/usr/local/lib/python3.7/site-packages/requests/api.py", line 131, in put
    return request('put', url, data=data, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/adapters.py", line 498, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2019-05-09 15:54:09,535 - INFO - app.py - edit_rule - {"id": 61, "minRate": "1.3", "maxRate": "0.0", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "111", "maxStrideDays": 0, "banAllDays": 0, "banType": ["E90", "7M8", "73H", "333"], "transferBlack": ["AKL", "AKU", "AMS", "AOG", "ADL"], "carrierWhite": ["AF", "SV"], "carrierBlack": ["WS", "AF"], "type": 0}
2019-05-09 15:54:09,539 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 15:57:35,557 - INFO - app.py - edit_rule - {"id": 61, "minRate": "2", "maxRate": "0.0", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "111", "maxStrideDays": 0, "banAllDays": 0, "banType": ["E90", "7M8", "73H", "333"], "transferBlack": ["AKL", "AKU", "AMS", "AOG", "ADL"], "carrierWhite": ["AF", "SV"], "carrierBlack": ["WS", "AF"], "type": 0}
2019-05-09 15:57:35,558 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 16:19:17,839 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-09 16:19:52,626 - INFO - app.py - edit_rule - {"id": 61, "minRate": "2.0", "maxRate": "3.0", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "111", "maxStrideDays": 0, "banAllDays": 0, "banType": ["E90", "7M8", "73H", "333"], "transferBlack": ["AKL", "AKU", "AMS", "AOG", "ADL"], "carrierWhite": ["AF", "SV"], "carrierBlack": ["WS", "AF"], "type": 0}
2019-05-09 16:19:52,627 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 16:22:06,567 - INFO - app.py - edit_rule - {"id": 62, "minRate": "1.2", "maxRate": "2", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 16:22:06,572 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 16:23:42,670 - INFO - app.py - edit_rule - {"id": 62, "minRate": "1.2", "maxRate": "1.5", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 16:23:42,672 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 16:25:59,973 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-09 16:26:22,188 - INFO - app.py - edit_rule - {"id": 61, "minRate": "2.0", "maxRate": "4", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "111", "maxStrideDays": 0, "banAllDays": 0, "banType": ["E90", "7M8", "73H", "333"], "transferBlack": ["AKL", "AKU", "AMS", "AOG", "ADL"], "carrierWhite": ["AF", "SV"], "carrierBlack": ["WS", "AF"], "type": 0}
2019-05-09 16:26:22,189 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 16:31:13,576 - INFO - app.py - edit_rule - {"id": 62, "minRate": "1.2", "maxRate": "2", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 16:31:13,579 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 16:31:50,162 - INFO - app.py - edit_rule - {"id": 63, "minRate": "0.0", "maxRate": "1", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 16:31:50,162 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 16:32:46,786 - INFO - app.py - edit_rule - {"id": 62, "minRate": "1.2", "maxRate": "3", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 16:32:46,787 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 16:36:14,853 - INFO - app.py - edit_rule - {"id": 62, "minRate": "1.2", "maxRate": "2", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 16:36:14,854 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 16:41:59,839 - INFO - app.py - edit_rule - {"id": 61, "minRate": "2.0", "maxRate": "3", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "111", "maxStrideDays": 0, "banAllDays": 0, "banType": ["E90", "7M8", "73H", "333"], "transferBlack": ["AKL", "AKU", "AMS", "AOG", "ADL"], "carrierWhite": ["AF", "SV"], "carrierBlack": ["WS", "AF"], "type": 0}
2019-05-09 16:41:59,843 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 16:44:22,514 - INFO - app.py - edit_rule - {"id": 64, "minRate": "0.0", "maxRate": "2", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 16:44:22,517 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 16:49:24,430 - INFO - app.py - edit_rule - {"id": 63, "minRate": "0.0", "maxRate": "1.0", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 16:49:24,434 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 16:49:42,316 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-09 16:49:48,501 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-09 16:49:50,173 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-09 16:50:53,097 - INFO - app.py - edit_rule - {"id": 63, "minRate": "0.0", "maxRate": "2.5", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 16:50:53,103 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 16:51:02,831 - INFO - app.py - edit_rule - {"id": 61, "minRate": "2.0", "maxRate": "3.0", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "111", "maxStrideDays": 0, "banAllDays": 0, "banType": ["E90", "7M8", "73H", "333"], "transferBlack": ["AKL", "AKU", "AMS", "AOG", "ADL"], "carrierWhite": ["AF", "SV"], "carrierBlack": ["WS", "AF"], "type": 0}
2019-05-09 16:51:02,832 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 16:51:19,268 - INFO - app.py - edit_rule - {"id": 61, "minRate": "2.0", "maxRate": "3.0", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "111", "maxStrideDays": 0, "banAllDays": 0, "banType": ["E90", "7M8", "73H", "333"], "transferBlack": ["AKL", "AKU", "AMS", "AOG", "ADL"], "carrierWhite": ["AF", "SV"], "carrierBlack": ["WS", "AF"], "type": 0}
2019-05-09 16:51:19,269 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 16:55:37,589 - INFO - app.py - edit_rule - {"id": 64, "minRate": "0.0", "maxRate": "3", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 16:55:37,591 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 16:57:54,436 - INFO - app.py - edit_rule - {"id": 63, "minRate": "0.0", "maxRate": "1", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 16:57:54,441 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 17:19:08,156 - ERROR - app.py - log_exception - Exception on /edit_rule/65 [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/urllib3/connection.py", line 159, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw)
  File "/usr/local/lib/python3.7/site-packages/urllib3/util/connection.py", line 80, in create_connection
    raise err
  File "/usr/local/lib/python3.7/site-packages/urllib3/util/connection.py", line 70, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 61] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 354, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1229, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1275, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1224, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1016, in _send_output
    self.send(msg)
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 956, in send
    self.connect()
  File "/usr/local/lib/python3.7/site-packages/urllib3/connection.py", line 181, in connect
    conn = self._new_conn()
  File "/usr/local/lib/python3.7/site-packages/urllib3/connection.py", line 168, in _new_conn
    self, "Failed to establish a new connection: %s" % e)
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x10bbb01d0>: Failed to establish a new connection: [Errno 61] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/requests/adapters.py", line 449, in send
    timeout=timeout
  File "/usr/local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 638, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/usr/local/lib/python3.7/site-packages/urllib3/util/retry.py", line 399, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=8070): Max retries exceeded with url: /rule/doUpdateObject (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10bbb01d0>: Failed to establish a new connection: [Errno 61] Connection refused'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/xiyanzhi/workspace/myflaskapp/app.py", line 108, in wrap
    return f(*args, **kwargs)
  File "/Users/xiyanzhi/workspace/myflaskapp/app.py", line 264, in edit_rule
    r = requests.put(app.config['ENDPOINT']+app.config['UPDATE_URL'], headers=header, data=form)
  File "/usr/local/lib/python3.7/site-packages/requests/api.py", line 131, in put
    return request('put', url, data=data, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=8070): Max retries exceeded with url: /rule/doUpdateObject (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10bbb01d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
2019-05-09 17:19:56,259 - INFO - app.py - edit_rule - {"id": 63, "minRate": "0.0", "maxRate": "2.1", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 17:19:56,260 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 17:20:30,184 - INFO - app.py - edit_rule - {"id": 63, "minRate": "0.0", "maxRate": "2.1", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 17:20:30,186 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 17:28:14,266 - INFO - app.py - edit_rule - {"id": 64, "minRate": "0.0", "maxRate": "2.1", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 17:28:14,270 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 17:28:14,393 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-09 17:28:35,285 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-09 17:28:38,175 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-09 17:28:54,191 - INFO - app.py - edit_rule - {"id": 64, "minRate": "0.0", "maxRate": "2.2", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 17:28:54,193 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 17:30:06,672 - INFO - app.py - edit_rule - {"id": 63, "minRate": "0.0", "maxRate": "2.3", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 17:30:06,674 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 17:34:26,199 - INFO - app.py - edit_rule - {"id": 62, "minRate": "1.2", "maxRate": "2.9", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 17:34:26,200 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 17:35:26,337 - INFO - app.py - edit_rule - {"id": 66, "minRate": "0.0", "maxRate": "1", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 17:35:26,338 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 17:54:16,110 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-09 17:54:18,106 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-09 17:54:21,448 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-09 17:54:24,324 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-09 17:54:26,598 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-09 17:54:26,742 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-09 17:54:27,655 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-09 17:54:59,460 - INFO - app.py - edit_rule - {"id": 64, "minRate": "0.0", "maxRate": "2.9", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 17:54:59,461 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 17:59:13,870 - INFO - app.py - edit_rule - {"id": 62, "minRate": "1.2", "maxRate": "3", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 17:59:13,872 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 18:11:11,350 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-09 18:11:13,054 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-09 18:11:26,468 - INFO - app.py - edit_rule - {"id": 64, "minRate": "0.0", "maxRate": "1", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 18:11:26,469 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 18:15:27,135 - INFO - app.py - edit_rule - {"id": 65, "minRate": "0.0", "maxRate": "1", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "legTransferWhite": ["A-B:C", "D-E:F"], "type": 0}
2019-05-09 18:15:27,140 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 18:15:55,106 - INFO - app.py - edit_rule - {"id": 65, "minRate": "0.0", "maxRate": "2", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "legTransferWhite": ["A-B:C", "D-E:F"], "type": 0}
2019-05-09 18:15:55,107 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 18:16:16,357 - INFO - app.py - edit_rule - {"id": 65, "minRate": "0.0", "maxRate": "2.11", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "legTransferWhite": ["A-B:C", "D-E:F"], "type": 0}
2019-05-09 18:16:16,358 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 18:24:43,629 - INFO - app.py - edit_rule - {"id": 64, "minRate": "0.0", "maxRate": "3.1", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-09 18:24:43,632 - INFO - app.py - edit_rule - 修改规则成功
2019-05-09 18:30:09,148 - INFO - app.py - edit_rule - {"id": 65, "minRate": "0.0", "maxRate": "2.13", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "legTransferWhite": ["A-B:C", "D-E:F"], "type": 0}
2019-05-09 18:30:09,150 - INFO - app.py - edit_rule - 修改规则成功
2019-05-10 09:26:08,102 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-10 09:26:11,579 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-10 09:26:28,969 - INFO - app.py - edit_rule - {"id": 63, "minRate": "0.0", "maxRate": "2.2", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-10 09:26:28,971 - INFO - app.py - edit_rule - 修改规则成功
2019-05-10 09:28:46,110 - INFO - app.py - edit_rule - {"id": 63, "minRate": "0.0", "maxRate": "3", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-10 09:28:46,111 - INFO - app.py - edit_rule - 修改规则成功
2019-05-10 09:31:24,444 - INFO - app.py - edit_rule - {"id": 64, "minRate": "0.0", "maxRate": "3.2", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-10 09:31:24,445 - INFO - app.py - edit_rule - 修改规则成功
2019-05-10 09:34:11,052 - INFO - app.py - edit_rule - {"id": 65, "minRate": "0.0", "maxRate": "5", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "legTransferWhite": ["A-B:C", "D-E:F"], "type": 0}
2019-05-10 09:34:11,054 - INFO - app.py - edit_rule - 修改规则成功
2019-05-10 09:35:56,583 - INFO - app.py - edit_rule - {"id": 64, "minRate": "0.0", "maxRate": "5", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-10 09:35:56,584 - INFO - app.py - edit_rule - 修改规则成功
2019-05-10 09:44:01,665 - INFO - app.py - edit_rule - {"id": 65, "minRate": "0.0", "maxRate": "9.0", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "legTransferWhite": ["A-B:C", "D-E:F"], "type": 0}
2019-05-10 09:44:01,666 - INFO - app.py - edit_rule - 修改规则成功
2019-05-10 09:46:13,362 - INFO - app.py - edit_rule - {"id": 63, "minRate": "0.0", "maxRate": "5.5", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-10 09:46:13,365 - INFO - app.py - edit_rule - 修改规则成功
2019-05-10 09:47:52,348 - INFO - app.py - edit_rule - {"id": 64, "minRate": "0.0", "maxRate": "5.2", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "type": 0}
2019-05-10 09:47:52,350 - INFO - app.py - edit_rule - 修改规则成功
2019-05-10 10:52:59,140 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-10 10:53:00,575 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-10 10:53:01,549 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-10 10:53:02,478 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-10 10:53:03,473 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-10 10:53:11,936 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-10 10:54:22,864 - INFO - app.py - edit_rule - {"id": 65, "minRate": "0.0", "maxRate": "2", "maxFlightDistance": 0, "maxFlightTime": 0, "maxStopPoint": "0", "maxTransfer": "0", "maxTransferTime": "0", "maxStrideDays": 0, "banAllDays": 0, "legTransferWhite": ["A-B:C", "D-E:F"], "type": 0}
2019-05-10 10:54:22,866 - INFO - app.py - edit_rule - 修改规则成功
2019-05-27 14:33:14,924 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-27 14:52:46,879 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-27 14:59:01,542 - ERROR - app.py - dashboard - 连接数据库失败
2019-05-27 15:14:08,813 - INFO - app.py - add_rule - {"type": 0, "maxTransferTime": "111", "banAllDays": 0}
2019-05-27 15:14:08,814 - INFO - app.py - add_rule - 添加规则成功
